spec/migrator/migration_spec.cr

```diff
@@ -66,7 +66,7 @@ end
 
 describe Avram::Migrator::Migration::V1 do
   it "executes statements in a transaction" do
-    expect_raises Exception, %(relation "table_does_not_exist" does not exist) do
+    expect_raises Avram::FailedMigration do
```

Comment:
Can no longer assert on the message because it is in the cause, not in the expected error